src/lib/components/forms/files-input.svelte

```diff
@@ -0,0 +1,31 @@
+<script>
+  let files = null;
+
+  $: () => {
+    if (files) {
+      console.log(files);
+      for (const file of files) {
+        console.log(`${file.name}: ${file.size} bytes`);
+      }
+    }
+  };
+</script>
+
+<div class="flex items-center space-x-4">
+  <label
+    for="file-upload"
+    class="muted-red cursor-pointer rounded-lg px-4 py-2 text-white hover:bg-red-600"
+  >
+    Selecionar ficheiro
```

Comment:
This should be an input to the component so that we can place whatever text here!